messaging/__init__.py

```diff
@@ -14,7 +14,7 @@
 # sec_since_boot is faster, but allow to run standalone too
 try:
   from common.realtime import sec_since_boot
-except ImportError:
+except Exception:
```

Comment:
was this an issue? ideally, we only watch to catch ImportError

Reply:
Traceback (most recent call last):                                                                                                       
  File "./test_messaging.py", line 13, in <module>                                                                                       
    import cereal.messaging as messaging                                                                                                 
  File "/home/batman/openpilot/cereal/messaging/__init__.py", line 16, in <module>                
    from common.realtime import sec_since_boot                
  File "/home/batman/openpilot/common/realtime.py", line 8, in <module>                     
    from selfdrive.hardware import PC, TICI                                                                                              
  File "/home/batman/openpilot/selfdrive/hardware/__init__.py", line 5, in <module>   
    from selfdrive.hardware.eon.hardware import Android                                                                                  
  File "/home/batman/openpilot/selfdrive/hardware/eon/hardware.py", line 11, in <module>             
    NetworkType = log.ThermalData.NetworkType                                                                                            
AttributeError: module 'log.capnp' has no attribute 'ThermalData' 


was getting that

Reply:
that's because `selfdrive/hardware/eon/hardware.py` needs to be updated to the DeviceState struct

Reply:
we don't want to make this catch all exceptions, otherwise we'll never know if we break this unintentionally and always end up using the slower function

Reply:
I know, but we can't do that without merging this PR into OP first?

Reply:
the CI tests don't rely on openpilot, so those should be fine. if you check out a matching cereal cleanup openpilot branch, the tests should pass locally too

Reply:
Oh I thought this was broken in CI too, ok will revert